src/db/car/any.rs

```diff
@@ -124,6 +125,19 @@ where
     }
 }
 
+impl<ReaderT> PersistentStore for AnyCar<ReaderT>
```

Comment:
I feel it would be less error-prone to make `Blockstore` persistent by default and have a `GarbageCollectableStore` for storing blockchain data, what do you think?

Reply:
We can create an issue for that and we’ll address it as prioritized. It’s out of scope for this PR, I have already created an issue to generalize different stores a bit more to be able to rely on tests for different database types better.